src/components/LogIn/LogIn.jsx

```diff
@@ -0,0 +1,51 @@
+import * as React from "react"
+import { useRef, useState } from "react"
+import { Link, useNavigate } from "react-router-dom"
+import { Form, Button, Card } from "react-bootstrap" 
+import { useAuth } from "../../contexts/AuthContext"
+import "./LogIn.css"
+
+export default function LogIn() {
+    const emailRef = useRef()
+    const passwordRef = useRef()
+    const { login } = useAuth()
+    const [loading, setLoading] = useState(false)
+    const navigate = useNavigate()
+
+    const handleSubmit = async (e) => {
+        e.preventDefault()
+
+        try {
+            setLoading(true)
+            await login(emailRef.current.value, passwordRef.current.value)
+            navigate("/profile")
```

Comment:
Should it redirect to the profile page or the page they were previously on?

Reply:
I'm planning to have the user logged in before using the website, so once they sign in, they would always see the profile page (and can now click on the other pages)